.github/workflows/ci.yml

```diff
@@ -25,6 +25,8 @@ jobs:
             otp: 21.3.8.16
           - elixir: 1.10.3
             otp: 23.0.2
```

Comment:
```suggestion
          - elixir: 1.11.2
            otp: 21.3.8.16
```

Reply:
@josevalim i don't know if this will find the right Docker container built by Hex, let's see! :)